cmd/flowrunner/testdata/flows/all_actions.json

```diff
@@ -163,7 +163,7 @@
                         {
                             "uuid": "06153fbd-3e2c-413a-b0df-ed15d631835a",
                             "type": "call_webhook",
-                            "method": "get",
+                            "method": "GET",
```

Comment:
editor saves as uppercase so I think we can be strict here and require correct case